### ISSUE TITLE
Empty Options screen after unsuccessful AddCommand resumption

### ISSUE THREAD

**Issue by GetmanetsIrina** [smartdevicelink/sdl_hmi#433]
**Bug Report**
Empty Options screen after unsuccessful AddCommand resumption

**Preconditions:**
1. HMI and SDL are started
2. Mobile app is registered and activated
3. Add command with UI and VR parts are added

**Reproduction Steps:**
1. Set erroneous result code for UI interface of AddCommand RPC(HMI Settings->RPC control -> <App name>  )
2. Perform resumption
3. Open app Options

**Expected Behavior:**
Resumption is failed because of erroneous result code to AddCommand request. Options menu contains default buttons for different exits

**Observed Behavior:**
Resumption is failed because of erroneous result code to AddCommand request. Options menu is empty(see screenshot)
![image](https://user-images.githubusercontent.com/11210973/97183628-ad092600-17a6-11eb-82bd-69eefabed191.png)

and HMI sends
```
RPCClient.js:323 HMI -> SDL [15:28:18:63]: {"jsonrpc":"2.0","id":-10,"error":{"code":4,"message":"Erroneous response is assigned by settings","data":{}}}
UIRPC.js:1645 FFW.undefinedResponse
RPCClient.js:323 HMI -> SDL [15:28:18:64]: {"jsonrpc":"2.0","id":-10,"error":{"code":4,"message":"Erroneous response is assigned by settings","data":{}}}
UIRPC.js:1645 FFW.undefinedResponse
RPCClient.js:323 HMI -> SDL [15:28:18:65]: {"jsonrpc":"2.0","id":-10,"error":{"code":4,"message":"Erroneous response is assigned by settings","data":{}}}
UIRPC.js:1645 FFW.undefinedResponse
RPCClient.js:323 HMI -> SDL [15:28:18:66]: {"jsonrpc":"2.0","id":-10,"error":{"code":4,"message":"Erroneous response is assigned by settings","data":{}}}
UIRPC.js:1645 FFW.undefinedResponse
RPCClient.js:323 HMI -> SDL [15:28:18:68]: {"jsonrpc":"2.0","id":-10,"error":{"code":4,"message":"Erroneous response is assigned by settings","data":{}}}
```

**OS & Version Information**
OS: Ubuntu 18.04
SDL: tag 7.0.0-RC1 5c72cddc35
HMI: develop 727d0d3
SPT: 20201007 
HMI Log:  [hmi_log_empty_menu.txt](https://github.com/smartdevicelink/sdl_hmi/files/5439323/hmi_log_empty_menu.txt)

**Comment by GetmanetsIrina** [comment]
@theresalech Please note, this is a regression issue found by Luxoft during 7.0 RC1 release manual testing. Luxoft team is going to prepare a fix.

**Comment by AKalinich-Luxoft** [comment]
@theresalech please notice PR with fix is ready for Livio review

**Comment by theresalech** [comment]
@AKalinich-Luxoft at this time, we are focused on preparing the final Core 7.0 Release Candidate for Steering Committee approval. We will work to include this fix in the SDL HMI 5.4 release if time allows.
